modules/sonobiBidAdapter.js

```diff
@@ -60,7 +60,9 @@ export const spec = {
       payload.us = config.getConfig('userSync').syncsPerBidder;
     }
 
-    if (deepAccess(validBidRequests[0], 'crumbs.pubcid') || deepAccess(validBidRequests[0], 'params.hfa')) {
+    if (deepAccess(validBidRequests[0], 'userId.pubcid')) {
+      payload.hfa = `PRE-${validBidRequests[0].userId.pubcid}`;
+    } else if (deepAccess(validBidRequests[0], 'crumbs.pubcid') || deepAccess(validBidRequests[0], 'params.hfa')) {
       payload.hfa = deepAccess(validBidRequests[0], 'params.hfa') ? deepAccess(validBidRequests[0], 'params.hfa') : `PRE-${deepAccess(validBidRequests[0], 'crumbs.pubcid')}`;
```

Comment:
they prefer `params.hfa` over `crumbs.pubcid`.  are we sure they don't want to prioritize that over `userId.pubcid` as well?